lib/scarpe/wv/background.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
+require_relative "../base64"
```

Comment:
I'd prefer require "scarpe/base64" here since at some point we want to separate the core Shoes API into a different gem than Webview.